### PR TITLE
answers are written into file

### PR DESCRIPTION
**Pull request by MiriyalaM**
Hey Bhawana! Please review my answers.

### CODE REVIEW

File: answers.txt [hunk 1]
```diff
@@ -1,10 +1,12 @@
 1. What is a gitignore file and what should it include for a Java project?
+  a) gitignore file is a text document which contains a list of files that you want git to ignore in your work directory. ".gitignore" tells git which files (or patterns) it should ignore.
+  b) It should include ".mtj.tmp/" ( which is Mobile Tools for Java (J2ME)) for a java project.
```
Comment (BhawanaPd) [inline answers.txt]: Please include everything that a Java project should include.

**Review by BhawanaPd — APPROVED** [review]

File: answers.txt (hunk 1 above)
Comment (MiriyalaM) [reply answers.txt]: Sure. Thank you.

File: answers.txt [hunk 2]
```diff
@@ -1,10 +1,21 @@
 1. What is a gitignore file and what should it include for a Java project?
```
Comment (shashidharvanteru) [inline answers.txt:1]: everthing looks perfect

File: answers.txt [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
+  a) gitignore file is a text document which contains a list of files that you want git to ignore in your work directory. ".gitignore" tells git which files (or patterns) it should ignore.
```
Comment (shashidharvanteru) [inline answers.txt:2]: everything looks perfect

**Pull request merged**